_config.yml

```diff
@@ -14,18 +14,11 @@ execute:
   allow_errors              : false # If `False`, when a code cell raises an error the execution is stopped, otherwise all cells are always run.
   stderr_output             : show  # One of 'show', 'remove', 'remove-warn', 'warn', 'error', 'severe'
 
-# Information about where the book exists on the web
-repository:
-  url: https://bayesiancomputationbook.com  # Online location of your book
-  # path_to_book: docs  # Optional path to your book, relative to the repository root
-  branch: gh-pages  # Which branch of the repository should be used when creating links (optional)
```

Comment:
I think we also might need this or the site breaks maybe. Once we merge Im sure we'll figure it out fast